daemons/lib-copy-offload/test-tool/main.c

```diff
@@ -51,12 +51,17 @@ void usage(const char **argv) {
     fprintf(stderr, "       -m SLOTS           Number of slots (processes).\n");
     fprintf(stderr, "       -M MAX_SLOTS       Maximum number of slots (processes).\n");
     fprintf(stderr, "       -d                 Perform a dry run.\n");
-    fprintf(stderr, "       -C MY_HOSTNAME     Name of the host submitting the request. (for development/debugging)\n");
+    fprintf(stderr, "\n");
+    fprintf(stderr, "Usage: %s [COMMON_ARGS] -S <ARGS>\n", argv[0]);
+    fprintf(stderr, "    -Q            Submit a status request for the specified job.\n");
```

Comment:
The other CRUD options are all lower case (-o, -l, -c). I think this should be -q

Reply:
done